resources/helm/dask-gateway/values.yaml

```diff
@@ -234,6 +234,9 @@ controller:
 # that is acting as a proxy for traffic towards the gateway or user created
 # DaskCluster resources.
 traefik:
+  # If traefik is already installed in the cluster, we do not need to install traefik
```

Comment:
This commend for example doesn't clarify the CRD complexity, I figure its worth being explicit and saying that this toggle impacts the non-CRD resources for traefik deployed by this helm chart, but that the CRDs needs to be opt-out via `--skip-crd` during installation.

Reply:
I changed the comment here to include crd information.